packages/fluentui/docs/src/examples/components/Form/Rtl/FormExample.rtl.tsx

```diff
@@ -23,7 +23,7 @@ const FormExampleRtl = () => (
         showSuccessIndicator: false,
       }}
     />
-    <Form.Field label="أوافق على الشروط" control={{ as: 'input' }} type="checkbox" id="conditions" />
+    <Form.Field control={{ as: Checkbox, label: 'أوافق على الشروط' }} id="conditions" />
```

Comment:
Maybe good moment to replace with the Form Components

```suggestion
    <Form.Checkbox  label=" 'أوافق على الشروط'" id="conditions" />
```

Reply:
This pattern can be replaced similarly across the files in the PR